content/webapp/pages/season.tsx

```diff
@@ -108,6 +112,9 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     const booksQueryPromise = fetchBooks(client, {
       predicates: [`[at(my.books.seasons.season, "${id}")]`],
     });
+    const articlesQueryPromise = fetchArticles(client, {
+      predicates: [`[at(my.articles.seasons.season, "${id}")]`],
```

Comment:
Not something to address in this PR, but I wonder if we should be adding a check that this really is a Prismic ID?

i.e. somebody isn't putting in garbage with extra quote marks to run arbitrary queries

(We may be handling this elsewhere, in which case we should leave a comment here.)